requirements.txt

```diff
@@ -1,5 +1,7 @@
 geopy>=1.11.0
+numpy==1.13.1
+# Change order of eggs to build wheel in docker image
```

Comment:
@caduvieira I was talking with @cuducos earlier and we were thinking about changing the comment to something like `# scipy must come before scikitlearn` what do you think?

Reply:
That's a more clear comment. Ok